Sources_v3/StreamChat/ChatClient.swift

```diff
@@ -304,6 +305,13 @@ public class _ChatClient<ExtraData: ExtraDataTypes> {
             builder(self.databaseContainer, self.eventNotificationCenter, self.apiClient)
         }
     }
+    
+    /// Before CIS-555, `webSocketClient` was initialized in `createBackgroundWorkers` in `init` since it was passed
+    /// to the background workers. Now, background workers do not access `webSocketClient` and it's not being
+    /// initialized in `init`, so we initialize it here explicitly.
+    private func createWebSocketClient() {
+        _ = webSocketClient
+    }
```

Comment:
I don't get why we have the comment here. Can't we just do it the way such that it doesn't raise any questions and doesn't require explaining comments? :)